aspnetcore/fundamentals/environments.md

```diff
@@ -242,13 +242,121 @@ See [Configuration by environment](xref:fundamentals/configuration/index#configu
 
 ## Environment-based Startup class and methods
 
-When an ASP.NET Core app starts, the [Startup class](xref:fundamentals/startup) bootstraps the app. If a `Startup{EnvironmentName}` class exists, the class is called for that `EnvironmentName`:
+### Startup class conventions
 
-[!code-csharp[](environments/sample/EnvironmentsSample/StartupDev.cs?name=snippet&highlight=1)]
+When an ASP.NET Core app starts, the [Startup class](xref:fundamentals/startup) bootstraps the app. The app can define separate `Startup` classes for different environments (for example, `StartupDevelopment`), and the appropriate `Startup` class is selected at runtime. The class whose name suffix matches the current environment is prioritized. If a matching `Startup{EnvironmentName}` class isn't found, the `Startup` class is used.
```

Comment:
bootstrap is slang and doesn't MT. Make it a link or better yet don't use it. I think you're fine saying it load the app.